<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>298</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>421</width>
    <height>291</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>421</width>
    <height>298</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>灭了学生端3.1.0 正式版</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Documents and Settings/Administrator/桌面/未标题-2.bmp</normaloff>C:/Documents and Settings/Administrator/桌面/未标题-2.bmp</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>281</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>屏蔽教师机指令</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>教师机指令屏蔽（包括屏幕广播等）</string>
      </property>
      <widget class="QPushButton" name="thread_start">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>开始屏蔽</string>
       </property>
      </widget>
      <widget class="QToolButton" name="thread_stop">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>结束屏蔽</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="by_thread">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>16</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>线程限制法[推荐]</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="by_process">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>36</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>进程限制法</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="by_network">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>56</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>网络限制法</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>161</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>设置</string>
      </property>
      <widget class="QLineEdit" name="ProcessName">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>StudentMain.exe</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>学生端进程名：</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>401</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>状态</string>
      </property>
      <widget class="QLabel" name="status">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>381</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>16</y>
         <width>381</width>
         <height>23</height>
        </rect>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>401</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>强制窗口化</string>
      </property>
      <widget class="QPushButton" name="Windows_start">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>361</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>启动</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Windows_end">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>361</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>结束</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>关闭/启动/重启学生端</string>
     </attribute>
     <widget class="QPushButton" name="off_student">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>391</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>关闭</string>
      </property>
     </widget>
     <widget class="QPushButton" name="on_student">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>391</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>启动</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>401</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>鼠标手势</string>
      </property>
      <widget class="QComboBox" name="Action_Select">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>关闭学生端</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>启动学生端</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>重启学生端</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>预设选择：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="Pre_Setting">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>自定义</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>鼠标位于左边缘</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>鼠标位于右边缘</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>鼠标位于上边缘</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>鼠标位于下边缘</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>鼠标位于左上角</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>鼠标位于左下角</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>鼠标位于右上角</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>鼠标位于右下角</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="Mouse_Track_Start">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>82</y>
         <width>371</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>启动鼠标手势</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>当鼠标坐标满足</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Function_Edit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_2">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>50</y>
         <width>61</width>
         <height>22</height>
        </rect>
       </property>
       <property name="suffix">
        <string>ms</string>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>50</y>
         <width>54</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>延迟设置：</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="restart_student">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>391</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>重启</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>241</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>学生端程序路径：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="File_Select">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>100</y>
        <width>31</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>VPN</string>
     </attribute>
     <widget class="QPushButton" name="vpn_connect">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>391</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>连接VPN服务器</string>
      </property>
     </widget>
     <widget class="QPushButton" name="vpn_connect_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>391</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>断开VPN服务器</string>
      </property>
     </widget>
     <widget class="QPushButton" name="vpn_test">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>391</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ping测试</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>帮助</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>401</width>
        <height>241</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse</set>
      </property>
      <property name="buddy">
       <cstring></cstring>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>关于</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>401</width>
        <height>241</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;灭了学生端 3.1.0 正式版&lt;/span&gt;&lt;/p&gt;&lt;p&gt;基于Windows内核技术&lt;/p&gt;&lt;p&gt;作者：胡清阳（mmlmml1）&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
